gen/definitions/bgp_peer.yaml

```diff
@@ -63,6 +64,24 @@ attributes:
     description: 'Source Interface. Must match first field in the output of `show intf brief`.'
     default_value: 'unspecified'
     example: 'lo0'
+  - nxos_name: holdIntvl
+    tf_name: hold_time
+    type: Int64
+    description: 'BGP Hold Timer in seconds. The value ust be greater than the keep alive timer'
+    default_value: 180
+    example: 5
+    omit_empty_value: true
```

Comment:
What is the reason for using `omit_empty_value` here?

Reply:
I left that in there when I was testing. I was trying to see if that would remove the entry from the device config when it was removed from the HCL. 

I've noticed similar behavior to interface descriptions - if you add a description and deploy, and then remove it from the hcl and deploy the description will remain on the device. 

Do you know how to fix that behavior?

Reply:
Looking at the code it should push an empty description and therefore remove the description. If that is not the case, I would need to investigate further.